.github/workflows/generate-documentation-api.yml

```diff
@@ -0,0 +1,54 @@
+name: Generate Documentation API
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'src/api/public/api.ts'
```

Comment:
Why do we need to create a new file for the api, and not use **src/bpmn-visualization.ts** ?

Reply:
because there are some exports useless for the API user (i.e. newBpmnElementsRegistry)

like this, we ensure that we only document the API - that we want to expose

Reply:
I don't think it is the best way to do. Because, maybe we don't document it, but the user can access to those useless export.

I think we should fix the useless export.

Reply:
Can we merge this PR as a 1st step? We are not documented the whole public API for now even in declaration files via TS documentation.

Reply:
this is definetely true,
We have already started discussion on this topic yesterday with @tbouffard 

I created an issue so we can figure it out #1041